### DISCUSSION TITLE
TypeError while using Past Feat Dynamic Real Features

### DISCUSSION THREAD

**Discussion by sayedathar11** [awslabs/gluonts#1855] [Q&A]
I have a time series data which consists of multiple time series , the problem is with feat_static_cat , feat_dynamic_cat and feat_dynamic_real , the model was working fine and was being evaluated properly but when I added Past Feat Dynamic Real Features it was showing " TypeError: source_array must be array like object" , The model trained properly but was not being evaluated on Test Dataset .
I have also printed out the shape of test and train values of past_feat_dynamic_real .
![gts](https://user-images.githubusercontent.com/17905011/149132379-6f5f6dc3-b5ee-49e1-9b54-945d157ce730.PNG)
@mbohlkeschneider @lostella can you please help !

**Comment by hardik-raja-dsw** [comment]
I am also facing a similar issue during the evaluation of my model; Appreciate it if someone can provide any leads to this issue we are facing.
Thanks in advance.

**Reply by dmehrab06** [reply]
Hi, were you eventually able to resolve it?
